fix(sandbox): add timestamp to main log output

## Summary

Add timestamp to the sandbox main log output for better debugging and future telemetry upload.

Prerequisite for #460 (Sandbox telemetry collection)

## Problem

Current log format in `log.py` lacks timestamps:

```
[INFO] [sandbox:run-agent] Heartbeat sent
[ERROR] [sandbox:run-agent] Failed to execute Claude
```

This makes it difficult to:
- Correlate events across different log sources
- Debug timing-related issues
- Upload logs with proper timestamps to telemetry system

## Solution

Add ISO 8601 timestamp to log output:

```python
# Before
[INFO] [sandbox:run-agent] Heartbeat sent

# After
[2025-12-09T10:00:00Z] [INFO] [sandbox:run-agent] Heartbeat sent
```

## Implementation

Update `scripts/lib/log.py.ts`:

```python
from datetime import datetime, timezone

def _timestamp() -> str:
    return datetime.now(timezone.utc).strftime("%Y-%m-%dT%H:%M:%SZ")

def log_info(msg: str) -> None:
    print(f"[{_timestamp()}] [INFO] [sandbox:{SCRIPT_NAME}] {msg}", file=sys.stderr)
```

## Files to Modify

- `turbo/apps/web/src/lib/e2b/scripts/lib/log.py.ts`

## Acceptance Criteria

- [ ] All log functions include ISO 8601 timestamp
- [ ] Timestamp is in UTC timezone
- [ ] Existing tests pass

---
🤖 Generated from conversation with [Claude Code](https://claude.com/claude-code)

--- a/turbo/apps/web/src/lib/e2b/scripts/lib/log.py.ts
+++ b/turbo/apps/web/src/lib/e2b/scripts/lib/log.py.ts
@@ -5,33 +5,39 @@
 export const LOG_SCRIPT = `#!/usr/bin/env python3
 """
 Unified logging functions for VM0 agent scripts.
-Format: [LEVEL] [sandbox:SCRIPT_NAME] message
+Format: [TIMESTAMP] [LEVEL] [sandbox:SCRIPT_NAME] message
 """
 import os
 import sys
+from datetime import datetime, timezone
 
 # Default script name, can be overridden by setting LOG_SCRIPT_NAME env var
 SCRIPT_NAME = os.environ.get("LOG_SCRIPT_NAME", "run-agent")
 DEBUG_MODE = os.environ.get("VM0_DEBUG", "") == "1"
 
 
+def _timestamp() -> str:
+    """Get current UTC timestamp in ISO 8601 format."""
+    return datetime.now(timezone.utc).strftime("%Y-%m-%dT%H:%M:%SZ")
+
+
 def log_info(msg: str) -> None:
     """Log info message to stderr."""
-    print(f"[INFO] [sandbox:{SCRIPT_NAME}] {msg}", file=sys.stderr)
+    print(f"[{_timestamp()}] [INFO] [sandbox:{SCRIPT_NAME}] {msg}", file=sys.stderr)
 
 
 def log_warn(msg: str) -> None:
     """Log warning message to stderr."""
-    print(f"[WARN] [sandbox:{SCRIPT_NAME}] {msg}", file=sys.stderr)
+    print(f"[{_timestamp()}] [WARN] [sandbox:{SCRIPT_NAME}] {msg}", file=sys.stderr)
 
 
 def log_error(msg: str) -> None:
     """Log error message to stderr."""
-    print(f"[ERROR] [sandbox:{SCRIPT_NAME}] {msg}", file=sys.stderr)
+    print(f"[{_timestamp()}] [ERROR] [sandbox:{SCRIPT_NAME}] {msg}", file=sys.stderr)
 
 
 def log_debug(msg: str) -> None:
     """Log debug message to stderr (only if VM0_DEBUG=1)."""
     if DEBUG_MODE:
-        print(f"[DEBUG] [sandbox:{SCRIPT_NAME}] {msg}", file=sys.stderr)
+        print(f"[{_timestamp()}] [DEBUG] [sandbox:{SCRIPT_NAME}] {msg}", file=sys.stderr)
 `;

